2022-03-27 09:02:53,630 INFO    MainThread:1047 [wandb_setup.py:_flush():69] setting env: {}
2022-03-27 09:02:53,630 INFO    MainThread:1047 [wandb_setup.py:_flush():69] setting login settings: {}
2022-03-27 09:02:53,630 INFO    MainThread:1047 [wandb_init.py:_log_setup():336] Logging user logs to /workspace/graph-neural-pde/src/wandb/run-20220327_090253-vanilla alpha_ = 0.9 alpha = default dopri5 test1/logs/debug.log
2022-03-27 09:02:53,630 INFO    MainThread:1047 [wandb_init.py:_log_setup():337] Logging internal logs to /workspace/graph-neural-pde/src/wandb/run-20220327_090253-vanilla alpha_ = 0.9 alpha = default dopri5 test1/logs/debug-internal.log
2022-03-27 09:02:53,630 INFO    MainThread:1047 [wandb_init.py:init():369] calling init triggers
2022-03-27 09:02:53,630 INFO    MainThread:1047 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2022-03-27 09:02:53,630 INFO    MainThread:1047 [wandb_init.py:init():418] starting backend
2022-03-27 09:02:53,630 INFO    MainThread:1047 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-27 09:02:53,645 INFO    MainThread:1047 [backend.py:ensure_launched():123] starting backend process...
2022-03-27 09:02:53,659 INFO    MainThread:1047 [backend.py:ensure_launched():128] started backend process with pid: 1073
2022-03-27 09:02:53,661 INFO    MainThread:1047 [wandb_init.py:init():423] backend started and connected
2022-03-27 09:02:53,662 INFO    MainThread:1047 [wandb_init.py:init():465] updated telemetry
2022-03-27 09:02:53,663 INFO    MainThread:1047 [wandb_init.py:init():484] communicating current version
2022-03-27 09:02:54,809 INFO    MainThread:1047 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-27 09:02:54,810 INFO    MainThread:1047 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2022-03-27 09:02:55,171 INFO    MainThread:1047 [wandb_init.py:init():522] starting run threads in backend
2022-03-27 09:02:58,086 INFO    MainThread:1047 [wandb_run.py:_console_start():1524] atexit reg
2022-03-27 09:02:58,087 INFO    MainThread:1047 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2022-03-27 09:02:58,090 INFO    MainThread:1047 [wandb_run.py:_redirect():1376] Redirecting console.
2022-03-27 09:02:58,093 INFO    MainThread:1047 [wandb_run.py:_redirect():1438] Redirects installed.
2022-03-27 09:02:58,093 INFO    MainThread:1047 [wandb_init.py:init():546] run started, returning control to user process
2022-03-27 09:02:59,123 INFO    MainThread:1047 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2022-03-27 09:02:59,126 INFO    MainThread:1047 [wandb_run.py:_restore():1466] restore
2022-03-27 09:03:01,541 INFO    MainThread:1047 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 900
}

2022-03-27 09:03:02,195 INFO    MainThread:1047 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 6986
}

2022-03-27 09:03:02,749 INFO    MainThread:1047 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6986
  total_bytes: 6986
}

2022-03-27 09:03:02,851 INFO    MainThread:1047 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6986
  total_bytes: 6986
}

2022-03-27 09:03:02,956 INFO    MainThread:1047 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6986
  total_bytes: 6986
}

2022-03-27 09:03:03,065 INFO    MainThread:1047 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6986
  total_bytes: 6986
}

2022-03-27 09:03:03,167 INFO    MainThread:1047 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6986
  total_bytes: 6986
}

2022-03-27 09:03:03,273 INFO    MainThread:1047 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6986
  total_bytes: 6986
}

2022-03-27 09:03:04,659 INFO    MainThread:1047 [wandb_run.py:_show_files():1838] logging synced files
